client/src/components/CopyUrlBox.js

```diff
@@ -1,29 +1,32 @@
 import styled from "styled-components";
 import Input from "./Input";
 import { FaRegCopy } from "react-icons/fa";
+import { useEffect } from "react";
```

Comment:
i don't think you used this in your file?

Reply:
fixed!